resources/contenttweaker/lang/en_us.lang

```diff
@@ -26,6 +26,9 @@ fluid.mithrillium=Molten Mithrillium
 fluid.adaminite=Molten Adaminite
 fluid.mithminite=Molten Mithminite
 
+base.material.heavy_metal=Heavy Metal
```

Comment:
Why all other fluids like `fluid.adaminite` not converted same way?

Reply:
Because the others use VanillaFactory.createFluid, while these two use MaterialSystem.getMaterialBuilder
https://github.com/Krutoy242/Enigmatica2Expert-Extended/blob/ae3823e472fd70961b4d5062068a2e7857cf94fb/scripts/cot/loader_cot.zs#L155

Reply:
Ok, understand.